core/src/main/resources/reference.conf

```diff
@@ -316,7 +316,6 @@ cassandra-journal {
   #  
   #    - enable clustering for your actor system
   #    - cassandra-journal.pubsub-notification=on              (send real-time announcements at most every sec)
-  #    - cassandra-query-journal.events-by-tag.eventual-consistency-delay = 0s
```

Comment:
Ah @patriknw it is here, i did push it

Reply:
good, I missed it. So, the benefit of pubsub would be that one can have a longer poll-interval and anyway (most often) get the result as quickly as the `eventual-consistency-delay`. Compared to eventual-consistency-delay + poll-interval  (poll-interval/2 on average)

Reply:
Yup the refresh interval defaults to 3s so if a new event comes just after that it could be nearly 3s before you get it